Makefile

```diff
@@ -43,7 +43,7 @@ lint:
 	pylint -rn --ignore=gauopen qiskit/aqua qiskit/chemistry qiskit/finance qiskit/ml qiskit/optimization test tools
 
 style:
-	pycodestyle --max-line-length=100 --exclude=gauopen qiskit/aqua qiskit/chemistry qiskit/finance qiskit/ml qiskit/optimization test tools
```

Comment:
Why were these options removed? Does `pycodestyle` get this info from somewhere else?

Reply:
Yes, in order to let pycodestye accept the new class named ‘I’,  this error had to be ignored which disables the default configuration that ignores a few other errors. Adding all those parameters to the command line makes it too long, so now qiskit-aqua follows qiskit-ignis’s example and has pycodestyle config. parameters in tox.ini